test/runtests.jl

```diff
@@ -1,10 +1,12 @@
 using Test
-
-import SimpleBenders
+include("../src/SimpleBenders.jl")
```

Comment:
These two lines are not needed. To run tests, check the docs on using the Julia package manager on docs.julialang.org
(I'm on my phone and can't find the proper link)

Reply:
Sure. I assume you are suggesting that I should test the scripts within the project environment.  I have checked the usage of `Pkg` and removed the two lines.